app/views/content/is-teaching-right-for-me/physics.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Teach physics
+description: |-
+    Find out how to become a physics teacher, including what you'll be teaching and what funding is available to help you train.
+subcategory: What to teach
+navigation: 5.42
```

Comment:
Shouldn't maths come before physics - ie alphabetically?

NB once you add more subject cards here, this numbering is not going to work as you won't have any/enough gaps into which to slot other pages. Given we have other subject pages planned for relatively soon, I'd suggest you use the following numbering for now:

computing - 5.43
maths - 5.46
physics - 5.48
all other subjects - 5.49

This will allow us to fit in the next few subject pages easily - we will then have think more broadly about how we rejig the numbering of all the sections below this one